src/Command/PrivatizeConstantsCommand.php

```diff
@@ -64,7 +64,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $sources = (array) $input->getArgument('sources');
         $excludedPaths = (array) $input->getOption('exclude-path');
 
-        $phpFileInfos = PhpFilesFinder::find($sources, $excludedPaths);
+        $phpFileInfos = PhpFilesFinder::find(
+            $sources,
+            $excludedPaths,
+            static fn (SplFileInfo $splFileInfo): bool => ! str_ends_with(
+                $splFileInfo->getFilenameWithoutExtension(),
+                'Interface'
```

Comment:
What about interface that do not have such suffix?

Reply:
Sorry, do not get your idea.

Reply:
This interface should be skipped too, but the there is no "Interface" suffix:

```php
interface NotSuffixed
{
}
```

Reply:
Parse file content for this check will be ok?

Reply:
Indeed, that's the way.

Somewhere here maybe: https://github.com/rectorphp/swiss-knife/blob/b2c4ffe818c3fa329ba4df27e490c7812a8fddce/src/PHPStan/ClassConstantResultAnalyser.php#L39

Reply:
I have done it another way. It does not cover all edge cases, but it should work for 99% of cases.